cla/signing-tutorial.md

```diff
@@ -0,0 +1,24 @@
+# CoatRack CLA Signing Tutorial
+
+When submitting a pull request for the first time, you will need to agree to the contributor license agreement (CLA). For example single, private persons must agree on the [CLA for individuals](https://github.com/coatrack/cla/latest/individual-cla.pdf) and organizations must agree on the [CLA for entities](https://github.com/coatrack/cla/latest/entity-cla.pdf). To do this, copy, rename and move [signing-template.md](https://github.com/coatrack/cla/signing-template.md) to `/cla/contributors/<your-github-username>.md`. Fill in the missing information, including the SHA-256 checksum of the corresponding CLA just mentioned, and add this file to your first pull request.
+
+
+
+## Structure of the 'cla' folder
+
+For transparency reasons, we will briefly explain here how our CLA system works. Harmony provides [templates](http://harmonyagreements.org/agreements.html) for CLA's in ODT format which are stored in the [original](https://github.com/coatrack/cla/source/original) folder. Based on that, we made some customizations fitting our requirements which are stored in the [customized](https://github.com/coatrack/cla/source/customized) folder. With LibreOffice, for example, you can easily see the differences between the original and our customized version by comparing the two ODT files. The customized CLA ODT's are converted to PDF's and stored in the [latest](https://github.com/coatrack/cla/latest) folder. These PDF files are the ones that have to be signed/hashed in order to be allowed to contribute to CoatRack.
+
+
+
+## License
```

Comment:
I would put this information _before_ the signing instructions.

Reply:
Done.